package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-api",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Is it a breaking change? Should we increase the version in the cli as well? >1.2.3